google-api-client/pom.xml

```diff
@@ -127,10 +128,20 @@
       <artifactId>commons-codec</artifactId>
       <scope>provided</scope>
       <version>1.6</version>
-   </dependency>
+    </dependency>
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava-jdk5</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.json</groupId>
+      <artifactId>javax.json-api</artifactId>
+      <version>1.0</version>
```

Comment:
The dependencies with versions normally go in the parent's pom.xml, and then the version isn't specified here. That guarantees that all the projects use the same version of dependencies.